scripts/pysa_fuzzer/fuzzer.py

```diff
@@ -0,0 +1,130 @@
+import random
+import textwrap
+from typing import List, Set, Callable
+
+class CodeGenerator:
+    def __init__(self):
+        self.sources = ['user_input()', 'get_data()', 'fetch_value()']
+        self.sinks = ['print', 'send_data', 'log_output']
+        self.variables = [chr(i) for i in range(ord('a'), ord('z') + 1)]
+        self.defined_variables: Set[str] = set()
+        self.current_var_index = 0
+
+    def indent(self, text: str, spaces: int = 4) -> str:
+        return textwrap.indent(text, ' ' * spaces)
+
+    def generate_arithmetic_expression(self, depth: int) -> str:
+        if depth == 0:
+            return str(random.randint(0, 10))
+        op = random.choice(['+', '-', '*', '/'])
+        return f"({self.generate_arithmetic_expression(depth - 1)} {op} {self.generate_arithmetic_expression(depth - 1)})"

```

Comment:
This is still not truly recursive. This should call `generate_expression`.
same thing below.